app/services/api/webflow-api.ts

```diff
@@ -122,6 +122,7 @@ export const useScheduleScreenData = () => {
     isRefetching,
     schedules: [
       {
+        bannerTx: "scheduleScreen.workshopBanner",
```

Comment:
great idea. do we need to update the types somewhere also?

Reply:
This is of type `Schedule`. I've adjusted that type above for the optional `bannerTx`.

Reply:
ahh missed it, sorry.